locales/base/translation.json

```diff
@@ -1405,6 +1405,26 @@
     "estimatedAmountTitle": "Estimated Amount",
     "estimatedAmountBody": "Conversions take place on a third party decentralized exchange that finds you the best rate for your swap. \n\n Your trade will be completed at a price within {{slippagePercent}}% of the estimate or the transaction will be cancelled & funds returned to your wallet."
   },
+  "swapCompleteScreen": {
+    "swapPending": "Swap Pending",
+    "swapSuccess": "Swap Successful!",
+    "exchangeRateSubtext": "Finding the best exchange rate...",
+    "approvingSubtext": "Approving funds...",
+    "completingSubtext": "Completing swap...",
+    "swapAgain": "Swap Again",
+    "swapPriceModal": {
+      "title": "Prices have changed",
+      "action": "Continue",
+      "body1": "Crypto amounts & fees can change due to due to fluctuations in the crypto market.",
```

Comment:
instead of body1 and body2, can we use `\n` inside 1 string? there are a few examples of this in the translation file. (we should avoid putting non descriptive translation keys like key1 and key2, as the names are useful for translators when providing translations, since the the key name is the only context they have for where it's used. the more descriptive the name, the better quality translations we will get)

Reply:
Fixed in ed092f282cd6e383afffbc23773b561187406949!